submitter/relayer/relayer.go

```diff
@@ -80,9 +80,15 @@ func (rl *Relayer) SendCheckpointToBTC(ckpt *ckpttypes.RawCheckpointWithMetaResp
 		return nil
 	}
 
-	if rl.lastSubmittedCheckpoint == nil || rl.lastSubmittedCheckpoint.Epoch < ckptEpoch {
+	if rl.lastSubmittedCheckpoint == nil ||
+		rl.lastSubmittedCheckpoint.Tx1 == nil ||
+		rl.lastSubmittedCheckpoint.Epoch < ckptEpoch {
 		rl.logger.Infof("Submitting a raw checkpoint for epoch %v for the first time", ckptEpoch)
 
+		if rl.lastSubmittedCheckpoint == nil {
```

Comment:
Can we separate the two cases? if `shouldSendCompleCheckpoint()` --> `convertCkptToTwoTxAndSubmit`, else if `shouldSendTx2()` --> `retrySendTx2`. I think this would be logically cleaner and some low-level funtions can be reused.